pysatMagVect/_core.py

```diff
@@ -346,41 +351,40 @@ def field_line_trace(init, date, direction, height, steps=None,
         Date to perform tracing on (year + day/365 + hours/24. + etc.)
         Accounts for leap year if datetime provided.
     direction : int
-         1 : field aligned, generally south to north. 
+         1 : field aligned, generally south to north.
         -1 : anti-field aligned, generally north to south.
     height : float
         Altitude to terminate trace, geodetic WGS84 (km)
     steps : array-like of ints or floats
-        Number of steps along field line when field line trace positions should 
+        Number of steps along field line when field line trace positions should
         be reported. By default, each step is reported; steps=np.arange(max_steps).
     max_steps : float
         Maximum number of steps along field line that should be taken
     step_size : float
         Distance in km for each large integration step. Multiple substeps
         are taken as determined by scipy.integrate.odeint
-        
+
     Returns
     -------
     numpy array
         2D array. [0,:] has the x,y,z location for initial point
         [:,0] is the x positions over the integration.
         Positions are reported in ECEF (km).
-        
-    
+
+
     """
-    
-    if recursive_loop_count is None:  
+
+    if recursive_loop_count is None:
         recursive_loop_count = 0
-    #     
+    #
     if steps is None:
         steps = np.arange(max_steps)
     if not isinstance(date, float):
         # recast from datetime to float, as required by IGRF12 code
         doy = (date - datetime.datetime(date.year,1,1)).days
         # number of days in year, works for leap years
         num_doy_year = (datetime.datetime(date.year+1,1,1) - datetime.datetime(date.year,1,1)).days
-        date = float(date.year) + \
-               (float(doy) + float(date.hour + date.minute/60. + date.second/3600.)/24.)/float(num_doy_year+1)
+        date = float(date.year) + float(doy)/float(num_doy_year) + float(date.hour + date.minute/60. + date.second/3600.)/24.
```

Comment:
I accepted this in the merge but it isn't correct. Old one is actually good. If DOY is normalized to 1 then time in a day has to be less than 1. End of year at end of last day, nearly num_doy_year+1

Reply:
There's something messed up in my fork.  I think I forked from master rather than develop.  Shouldn't be one of the changes I made.